package.json

```diff
@@ -92,11 +93,11 @@
     "zone.js": "^0.8.26"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^0.6.8",
+    "@angular-devkit/build-angular": "~0.13.0",
     "@angular-devkit/build-ng-packagr": "~0.6.8",
     "@angular/cli": "~6.0.8",
```

Comment:
should we not also just upgrade the cli too?

Reply:
We probably can at some point, but wanted to keep the changeset as small as possible for now. CLI doesn't have direct impact on the code itself, more around scaffolding and all